documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -65,6 +72,21 @@ group.id=my-group-id <1>
 ----
 <1> Add a consumer to a consumer group using a group id.
 
+== Choosing the right partition assignment strategy
+
+Select an appropriate partition assignment strategy, which determines how Kafka topic partitions are distributed among consumer instances in a group.
+
+Specify the strategy using the `partition.assignment.strategy` consumer configuration property. 
+The *range* assignment strategy assigns a range of partitions to each consumer, and is useful when you want to process related data together.
+
+Alternatively, opt for a *round robin* assignment strategy for equal partition distribution among consumers, which is ideal for high-throughput scenarios requiring parallel processing.
+
+For more stable partition assignments, consider the *sticky* and *cooperative sticky* strategies. 
+Sticky strategies maintain assigned partitions during rebalances, when possible. 
+The cooperative sticky strategy also supports cooperative rebalances, enabling uninterrupted consumption from partitions that are not reassigned.
```

Comment:
Based on what do they maintain the stickiness?

Reply:
I updated to mention that stickiness is based on previously assigned partitions.

> For more stable partition assignments, consider the *sticky* and *cooperative sticky* strategies. 
> Sticky strategies maintain assigned partitions during rebalances, when possible. 
> If a consumer was previously assigned certain partitions, the sticky strategies aim to reassign those same partitions back to the same consumer after a rebalance.
> Leaving partition assignments in place reduces the overhead on partition movements.
> The cooperative sticky strategy also supports cooperative rebalances, enabling uninterrupted consumption from partitions that are not reassigned.